docs/ABI.rst

```diff
@@ -1039,7 +1039,7 @@ Types
   type-list ::= list-type '_' list-type*     // list of types
   type-list ::= empty-list
 
-  list-type ::= type identifier? 'z'? 'd'?   // type with optional label, inout convention and variadic specifier
+  list-type ::= type identifier? 'z'? 'h'? 'd'?   // type with optional label, inout convention, shared convention, and variadic specifier
```

Comment:
Spoke to @eeckstein in person.  We agreed to try the single-letter mangling and revisit a two-char mangling at a later date.